src/components/CustomClient.js

```diff
@@ -1,66 +1,74 @@
 import React from 'react';
-import { Divider, Form, Message } from 'semantic-ui-react';
+import { Form, Message } from 'semantic-ui-react';
+
 
 /**
- * Load previous session from object.
- * @callback loadClient
- * @param {object} clientContainer - Object containing previous session as `client`.
+ * Propagate data from JSON object.
+ * @callback load
+ * @param {object} toLoad - Parsed JSON object to use.
+ * @todo Change to just 'load'? Change this component
+ *     to a more generic `<Load>`.
```

Comment:
I don't think the callback documentation is local to this class, so just `load` may not be clear enough (no matter where you define the callback it can be referenced by any other jsdoc and gets its own entry in the toc). I do agree about changing the name `<CustomClient>`. That one never made sense to me.

Reply:
Hmm, so we'll have to think about that. It may not be a change for this PR to make.

How about something about the input type? `<AcceptJSON>`, `<LoadJSON>`, `<GetJSON>`. Basically, it has an input to get a user string, it only accepts JSON, and then it sends the data to the parent. `GetUserJSON`? `JSONInput`? `JsonImport`?

Reply:
Just "JSON" seems too generic to me. How about `<ClientJSONImporter>`? Or `<ClientImporter>`?

Reply:
Well, I'm thinking of making it more generic - the way it's built, it could import any data, not just client stuff.

Reply:
OK. `<JSONImporter>`? I think it should be a noun, and "Input" seems wrong because that makes me think of an input on a form, while this is a higher-level component.

Reply:
True, and that might work! I'll merge this for now anyway and we'll work on that later. @todo!